src/shared/components/palette/complex/write-complex-field.html

```diff
@@ -22,6 +22,7 @@ <h2 *ngIf="renderLabel" class="heading-h2">{{caseField | ccdFieldLabel}}</h2>
                        [contextFields]="caseFields"
                        [caseFields]="caseFields"
                        [formGroup]="formGroup"
+                       [formSubGroup]="complexGroup"
```

Comment:
Giri shall we call it `complexForm` rather than `formSubGroup`?

Reply:
May be complexFormGroup?

Reply:
ok :)